Expecto/Expecto.fs

```diff
@@ -704,6 +740,8 @@ module Impl =
       /// Number of parallel workers. Defaults to the number of
       /// logical processors.
       parallelWorkers : int
+      /// Run the tests randomly for the given TimeSpan.
+      stress : TimeSpan option
```

Comment:
Shouldn't this comment mention the parallelism? What does it "mean" to stress test?